types/gapi.drive.v3/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
Please do not include additional licences. all definitions in this repo are already distributed under the [MIT licence](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE)